project/src/main/webapp/polling_locations.html

```diff
@@ -0,0 +1,73 @@
+<!DOCTYPE html>
```

Comment:
Nit: Can you name the file with a hyphen instead of an underscore to match the other files in the front end?